### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by cooperlab**
Pull request for collaborative commenting and editing of documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,43 +1,33 @@
...
-* A: for patient _i_, the corresponding element in A is the index of the first patient in the at risk group of _i_. Look at Run.py for an example of how to calculate this vector using the provided functions. size:(number of patients, ).
+* Interpret a trained model and analyze feature importance (ModelAnalysis.py)
 
+The Run.py training module takes as input a python dictionary containing the following numpy arrays:
+* X: input data of size (number of patients, number of features). Patients must be sorted with respect to event or censoring times 'T'.
+* T: Time of event or time to last follow-up, appearing in increasing order and corresponding to the rows of 'X'. size: (number of patients, ).
+* O: Right-censoring status. A value of 1 means the event is observed (i.e. deceased or disease progression), a 0 value indicates that the sample is censored. size:(number of patients, ).
+* A: An index array encoding the at-risk set for each sample. For sample _i_, the _i_ th element in A is the index of the next sample in the at risk group of _i_ (see Run.py for an example of how to calculate this vector using the provided functions) size:(number of patients, ).
```
Comment (cooperlab) [inline README.md]: @safooray This is confusing - need to describe this better.

File: README.md [hunk 2]
```diff
@@ -1,43 +1,33 @@
...
-* X: input data of size (number of patients, number of features). Patients must be sorted with respect to T.
-* T: sorted survival labels; either time of event or time to last follow-up. size: (number of patients, ).
-* O: observed status. Array of 1s and 0s. 1 means event is observed, 0 means sample is censored. size:(number of patients, ). 
-Also sorted with respect to T.
-* A: for patient _i_, the corresponding element in A is the index of the first patient in the at risk group of _i_. Look at Run.py for an example of how to calculate this vector using the provided functions. size:(number of patients, ).
+* Interpret a trained model and analyze feature importance (ModelAnalysis.py)
 
+The Run.py training module takes as input a python dictionary containing the following numpy arrays:
+* X: input data of size (number of patients, number of features). Patients must be sorted with respect to event or censoring times 'T'.
+* T: Time of event or time to last follow-up, appearing in increasing order and corresponding to the rows of 'X'. size: (number of patients, ).
```
Comment (cooperlab) [inline README.md]: @safooray Order is increasing?

File: README.md [hunk 3]
```diff
@@ -1,43 +1,33 @@
...
 
-
-
-
-
-
-
-
-
+The provided example scripts assume the data is a .mat file containinig, 'Survival', 'Censored', and either 'Integ\_X' or 'Gene\_X' depending on what feature set we are using.
```
Comment (cooperlab) [inline README.md]: Need to clarify that this is a one-off for how we wrote the paper. We can add some commentary about using the TCGAIntegrator package to generate data.

File: README.md [hunk 4]
```diff
@@ -1,102 +1,64 @@
...
-For using the SurvivalNet Package with Docker, there is no need to write Dockerfile to pull the Docker Image.
-    
     sudo docker run -v /<hostmachine_data_path>/:/<container_data_path>/ -it cramraj8/survivalnet2.0 /bin/bash
-is enough to pull the Docker Image, run the container, and mount the host machine data directory with container data path.
-The container Data Path is usually be
-	***/SurvivalNet/data/<data_file_name>***
-  
+    
+will pull the Docker Image, run the container, and mount the host machine data directory with container data path.
+The container Data Path is ***/SurvivalNet/data/<data_file_name>***.
```
Comment (cooperlab) [inline README.md]: @cramraj8 please take a look at this. It seems clear that users have to mount a folder containing their data, but the details are a little unclear. We should add a bit more detail here.

File: README.md [hunk 5]
```diff
@@ -1,102 +1,64 @@
...
-If you want to run the example python script, you can navigate into
+Example python scripts can be found in the folder 
     
     cd /SurvivalNet/examples/ 
-folder.
-There you can find some python scripts. But *Run.py* is the proper python script that can tell you about the deep learning progress.
+
+where users can see how to train and validate deep survival models. The main script
     
     python Run.py
```
Comment (cooperlab) [inline README.md]: @cramraj8 Is Run.py in the examples folder? Are there other files in the examples folder?

File: README.md (hunk 4 above)
Comment (cramraj8) [reply README.md]: I will make some more explanation here.

Comment (cooperlab) [reply README.md]: I already made a pass at this - please review my version for accuracy.

**Pull request merged**